tests/test_transplants.py

```diff
@@ -143,6 +143,110 @@ def test_dryrun_reviewers_warns(client, db, phabdouble, auth0_mock):
     assert response.json["confirmation_token"] is not None
 
 
+def test_dryrun_codefreeze_warn(
+    client,
+    db,
+    phabdouble,
+    auth0_mock,
+    codefreeze_datetime,
+    monkeypatch,
+    request_mocker,
+):
+    product_details = "https://product-details.mozilla.org/1.0/firefox_versions.json"
+    request_mocker.register_uri(
+        "GET",
+        product_details,
+        json={
+            "NEXT_SOFTFREEZE_DATE": "freeze_date",
+            "NEXT_MERGE_DATE": "merge_date",
```

Comment:
If these are renamed to reflect the day relative to "today" (from the test's perspective), it will be easier to understand what this test is trying to do (i.e. in this case NEXT_SOFTFREEZE_DATE would say "two_days_ago", and "NEXT_MERGE_DATE" would say "tomorrow".)

Reply:
thank you for the re-work - i was trying to think of a way to make it more straightforward